docs/howtos/DataPlane Generation - DPG/06types.mdx

```diff
@@ -806,6 +793,110 @@ public final class Animal implements JsonSerializable<Animal> {
 </TabItem>
 </Tabs>
 
+### Nullable
+
+TypeSpec uses `| null` to represent nullable types. Nullability is handled differently in languages, but emitter authors will find information
+about nullability by inspecting the type of a property.
+
+<Tabs>
+<TabItem value="typespec" label="TypeSpec" default>
+
+```typespec
+model Foo {
+  basicNullableProperty: string | null;
+  modelNullableProperty: Bar | null;
+}
```

Comment:
How about the case `nullableProperty: A|B|null`? Will the returned type be 'union' or 'nullable'?

Reply:
`nullable` with value type of `union`.